src/databricks/labs/ucx/install.py

```diff
@@ -962,6 +963,21 @@ def validate_and_run(self, step: str):
         if not self.validate_step(step):
             self.run_workflow(step)
 
+    def _enable_files_in_repos(self):
+        # check if "enableProjectTypeInWorkspace" and "enableWorkspaceFilesystem" are set to false
+        project_type = self._ws.workspace_conf.get_status("enableProjectTypeInWorkspace")
```

Comment:
Supply comma-separated list of settings names to make just one api call.

Why do we need project type (git repos), by the way?

Reply:
During initial testing both value were needed. I tested against a new workspace and just flipping enableWorkspaceFilesystem suffices.